var/spack/repos/builtin/packages/py-resultsfile/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyResultsfile(PythonPackage):
+    """Python module to read output files of quantum chemistry programs"""
+
+    homepage = "https://gitlab.com/scemama/resultsFile"
+    url      = "https://gitlab.com/scemama/resultsFile/-/archive/v1.0/resultsFile-v1.0.tar.gz"
+
+    maintainers = ['scemama']
+
+    version('1.0', sha256='e029054b2727131da9684fa2ec9fb8b6a3225dc7f648216a9390267b2d5d60c3')
+
+    depends_on('python@2.7:2.8.999', type=('build', 'run'))
```

Comment:
It doesn't work with Python 3?

Reply:
Version 1.0 is for Python2 and version 2.0 is for Python3.
I updated the package file to provide for v1.0 for Python2 or v2.0 for Python3.